src/Microsoft.ML.OnnxTransform/OnnxCatalog.cs

```diff
@@ -11,23 +11,53 @@ public static class OnnxCatalog
     {
         /// <summary>
         /// Initializes a new instance of <see cref="OnnxScoringEstimator"/>.
```

Comment:
>Initializes a new instance of [](start = 12, length = 29)

Maybe switch to: Applies a pre-trained Onnx model. 

The current language is mostly used for constructors.